tests/test_client.py

```diff
@@ -222,5 +222,18 @@ def test_midnight_heartbeats(client, bucket):
     assert len(recv_events_after_midnight) == int(len(recv_events_merged) / 2)
 
 
+def test_filter_future_events(client, bucket):
```

Comment:
With the aw-core patch and not wanting to raise a exception but just do a log, remove this test.